zipkin/src/test/java/com/linecorp/armeria/client/tracing/HttpTracingClientTest.java

```diff
@@ -68,11 +67,7 @@ public void tearDown() {
 
     @Test
     public void newDecorator_shouldFailFastWhenRequestContextCurrentTraceContextNotConfigured() {
-        assertThatThrownBy(() -> HttpTracingClient.newDecorator(Tracing.newBuilder().build()))
-                .isInstanceOf(IllegalStateException.class).hasMessage(
-                "Tracing.currentTraceContext is not a RequestContextCurrentTraceContext scope. " +
-                "Please call Tracing.Builder.currentTraceContext(RequestContextCurrentTraceContext.INSTANCE)."
-        );
+        HttpTracingClient.newDecorator(Tracing.newBuilder().build());
```

Comment:
Thanks for adding this back. How about updating the test name as well? :-)

Reply:
Oops - fixed